openapi/paths/subscriptions@{id}@items@{itemId}.yaml

```
@@ -0,0 +1,41 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+  - name: itemId
+    in: path
+    description: ID of subscription item.
+    required: true
+    schema:
+      type: string
+put:
+  x-badge: Experimental
+  x-products:
+    - Core
+  tags:
+    - Orders
+  summary: Update subscription order item
```

suggestion: I think we should add 'a' to emphasize that this updates a singular order item.

We do try to keep summary descriptions short, and sometimes omit articles to do this, but I think it's useful here

```suggestion
  summary: Update a subscription order item
```